ci/build.yml

```diff
@@ -413,6 +413,15 @@ jobs:
     - template: tell-slack-failed.yml
     - template: report-end.yml
 
+- job: compatibility_stable_protobuf
```

Comment:
We might want to add this to `collect_build_data` which aggregates results. It doesn’t make a huge difference atm but at least for some slack notifications it makes more sense to wait on this.

Reply:
6d3da77

Reply:
@cocreature I'd resolve this if I did it right

Reply:
I'm not quite sure what @cocreature meant here. As it stands, what this will do is make this test part of the individual opt-in CI notification on #team-daml-ci, which as far as I'm aware nobody working on this uses.

If that's all you wanted out of this, hten yes, you did it right.